src/main/kotlin/org/objectionary/ddr/sources/Sources.kt

```diff
@@ -99,9 +102,10 @@ class SrsTransformed(
      *
      * @param filename path to current xmir file in source directory
      * @return path to the temporary directories
+     *
      */
     private fun generateTmpPath(filename: String): String {
-        val strPath = path.toString()
+        val strPath = inPath.toString()
```

Comment:
@c71n93 We are trying to avoid compound names. According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html